Sources/NIOFileSystem/ByteCount.swift

```diff
@@ -80,4 +80,22 @@ public struct ByteCount: Hashable, Sendable {
     }
 }
 
+extension ByteCount: AdditiveArithmetic {
+    public static var zero = ByteCount(bytes: 0)
```

Comment:
This should either be a `static let` or a `static` computed property:

```suggestion
    public static var zero: ByteCount {  ByteCount(bytes: 0) }
```

It's not a good idea to let any user change the meaning of `zero`!